content/docs/alerting/configuration.md

```diff
@@ -65,6 +65,8 @@ global:
   [ smtp_from: <tmpl_string> ]
   # The default SMTP smarthost used for sending emails.
   [ smtp_smarthost: <string> ]
+  # The default hostname to identify to the SMTP server.
```

Comment:
I think we should mention the default behaviour of using localhost here somehow.